src/screens/level.py

```diff
@@ -216,19 +217,17 @@ def create_particle(self, sprite):
     def apply_tool(self, tool: FarmingTool, pos, entity):
         match tool:
             case FarmingTool.AXE:
-                for tree in self.tree_sprites:
-                    if tree.rect.collidepoint(pos):
-                        tree.hit(entity)
-                        self.sounds['axe'].play()
+                trees_collided = pygame.sprite.spritecollide(entity, self.tree_sprites, False)
```

Comment:
I assume this is an intended collision change. I still want to mention that Mega_JC advised that PRs only fix one thing, in order for them to not become a mishmash of additions and making things difficult to track down if a revert is needed